GermanFilter/sections/antiadblock.txt

```diff
@@ -879,6 +879,8 @@ zappn.tv#%#!function(){window.somtag={cmd:function(){}};}();
 raidrush.net###ab
 raidrush.net#$##wrapfabtest { height: 1px !important; }
 raidrush.net#%#//scriptlet("prevent-setTimeout", "window.getComputedStyle(el)")
+! https://github.com/AdguardTeam/AdguardFilters/issues/80281
+nickles.de#%#//scriptlet('abort-current-inline-script', 'Nickles.Account')
```

Comment:
@stanislav-atr  It seems that this rule breaks website.
For example on `https://www.nickles.de/signup/`, "Community Livefeed" is not displayed.

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/114847018-86cf5500-9ddd-11eb-8edf-5122f364add3.png)

</details>